### ISSUE TITLE
How to calculate loss for validation ?

### ISSUE THREAD

**Issue by efedoganay** [yhenon/pytorch-retinanet#191]
I am training the code on a custom CSV dataset, it works fine but where and how should I change to calculate the validation loss?

**Comment by Nileshmaharjan** [comment]
@efedoganay  Have you found any solution for this?
